docker-hub/repos.md

```diff
@@ -122,15 +122,26 @@ repository to see its tags.
 
 ![Repository View](images/repos-create.png)
 
-![View Repo Tags](images/repo-overview.png)
+![View Repo Tags](images/repo-overview-tag-status.png)
```

Comment:
Given that the new image is replacing the old one, could you overwrite `repo-overview.png` with the new file instead? this image is only used on this page, so it's better to replace the old file instead of adding a new file with a different name.

```suggestion
![View Repo Tags](images/repo-overview.png)
```